docs/Administration.rst

```diff
@@ -3061,6 +3064,136 @@ this entry is updated:
 -  The number of added users is increased if some users were added to the bound group by an administrator
 
 
+.. _AdminAppCenter:
+
+============================
+Managing Application Center
+============================
+
+As a platform administrator, you can manage the Application Center by:
+
+-  Defining the number of allowed favorite applications
+-  Defining mandatory applications that appear in every favorite applications drawer/list
+-  Defining default icon for added applications
+-  Adding/Editing applications
+
+To access the administration menu for Application Center, you need to access to the left navigation menu through
+the hamburger icon -> Administration -> OTHER -> Applications
+
+|AC_image0|
```

Comment:
The label of the menu changed recently. Please update the image to reflect the correct label

Reply:
OK